public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
- [ ] Kindly remove this file and these other CRA files in the public directory: `robots.txt`, `logo512.png`, `logo192.png`.

  ![image](https://github.com/boniface193/math-magician/assets/54848363/5ba351f9-2b1e-44d8-88f6-6d28cc974265)